skeleton-typescript/build/bundles.js

```diff
@@ -29,8 +29,7 @@ module.exports = {
         "aurelia-logging-console",
         "bootstrap",
         "bootstrap/css/bootstrap.css!text",
-        "fetch",
-        "jquery"
```

Comment:
Actually, I removed it here as someone else added jquery in the index.html (it's being pulled from the CDN).